content/4_Gold/TopoSort.mdx

```diff
@@ -451,6 +451,49 @@ public class TopoSort {
 
 </JavaSection>
 
+<PySection>
+
+```python
+from collections import deque
+
+def main():
+	def bfs(n: int, G: List[List[int]]) -> List[int]:
```

Comment:
uh given that everything is in a function already maybe just take this out and put it right in `main`